gensim/test/test_keyedvectors.py

```diff
@@ -400,6 +400,43 @@ def test_load_model_and_vocab_file_ignore(self):
             model.get_vector(u'どういたしまして'), np.array([.1, .2, .3], dtype=np.float32)))
 
 
+try:
+    import keras  # noqa: F401
+
+    keras_installed = True
```

Comment:
```suggestion
    KERAS_INSTALLED = True
```

Reply:
I'm fine with the uppercase but this need to be changed in the rest of the file. Can this be done with this "commit suggestion" feature?

Reply:
No, unfortunately that part has to be done manually. Are you able to do it?

Reply:
Sure, I'll change it